beacon-chain/core/altair/epoch_precompute.go

```diff
@@ -346,3 +347,19 @@ func attestationDelta(
 
 	return reward, penalty, nil
 }
+
+// UnrealizedCheckpoints returns the justification and finalization checkpoints of the
```

Comment:
Who is going to call `UnrealizedCheckpoints`. It may be better to move `UnrealizedCheckpoints` under precompute pkg so we don't have to export `ProcessJustificationBits` and `ComputeCheckpoints `.  It feels like we are exporting many things that can be avoided

Reply:
This hasn't been decided it will depend on how hard it is to deprecate blockchain.store. the best candidate to call this is forkchoice and in this case I'd move this there but will still need to export those

Reply:
My recommendation: https://github.com/prysmaticlabs/prysm/commit/3f6cbbe0e5a72a4f58f26496000f0ba8cd89357d

Let's just move `UnrealizedCheckpoints` to `precompute` package. I don't see why it must be in `altair` package and what benefits that bring